giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/nepomuk/impl/NepomukUrlService.java

```diff
@@ -23,26 +23,33 @@
 
     @Autowired
     protected INepomukServiceDiscoverer nepomukDiscoverer;
-    
+
     @Autowired
     protected IPropertiesManager propertyManager;
-    
+
     @Autowired
     private ISystemMessageHandler messageHandler;
 
-    /* (non-Javadoc)
-     * @see edu.asu.diging.gilesecosystem.web.nepomuk.impl.INepomukUrlService#getFileDownloadPath(edu.asu.diging.gilesecosystem.web.domain.IFile)
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.diging.gilesecosystem.web.nepomuk.impl.INepomukUrlService#
+     * getFileDownloadPath(edu.asu.diging.gilesecosystem.web.domain.IFile)
      */
     @Override
     public String getFileDownloadPath(IFile file) {
         String nepomukUrl;
+        String downloadPath = "";
         try {
             nepomukUrl = nepomukDiscoverer.getRandomNepomukInstance();
         } catch (NoNepomukFoundException e) {
-            messageHandler.handleMessage("Could not download file. Nepomuk could not be reached.", e, MessageType.ERROR);
+            messageHandler.handleMessage("Could not download file. Nepomuk could not be reached.", e,
+                    MessageType.ERROR);
             return null;
         }
-        
-        return nepomukUrl + propertyManager.getProperty(Properties.NEPOMUK_FILES_ENDPOINT).replace("{0}", file.getStorageId());
+        downloadPath = nepomukUrl
+                + propertyManager.getProperty(Properties.NEPOMUK_FILES_ENDPOINT).replace("{0}", file.getStorageId());
+
+        return downloadPath;
```

Comment:
you actually didn't change anything here, did you? Just introduced a variable? revert that please.